### PR TITLE
Create Ransomware-readme.md

### PR DESCRIPTION
**Pull request by Michal-Vardi**
Ransomware pack article

## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Review by EliorKedar — APPROVED** [review]

**Comment by Michal-Vardi** [comment]
@glicht 
This ready to publish. Do I have to do anything else to close it?

**Comment by glicht** [comment]
> @glicht
> This ready to publish. Do I have to do anything else to close it?

You need to squash and merge: 

image

**Comment by glicht** [comment]
@Michal-Vardi what is the status here?

File: Ransomware-readme.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+For example, for Palo Alto Networks Cortex XDR the **users** and **hosts** fields should be mapped to XSOAR **Users** and **Hosts** fields, respectively. 
+
+## Testing the Pack
+After you install the pack and perform all required prerequisites, test the pack to make sure everything works. There are two ways to test this pack, either manually create an incident or ingest an incident of type **Post Intrusion Ransomware**.
+
+### Manual Test
+
+1. Create a new incident.
+2. Enter values for the **Users** and **Hosts** fields.
```
Comment (EliorKedar) [inline Ransomware-readme.md:99]: Remove this line and explain what will not work in the manual test

File: Ransomware-readme.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+3. Configure the **Post Intrusion Ransomware Investigation** playbook inputs.
+4. Use the sample ransom note available [here](#Sample-Ransom-Note).
+
+### Ingestion Test
+
+1. Configure an instance of the integration that you will use to ingest and create incidents of type **Post Intrusion Ransomware**.
+2. Map the corresponding fields from the integration to **Users** and **Hosts**.
```
Comment (EliorKedar) [inline Ransomware-readme.md:106]: Change the field name to 
1. Usernames
2. Hostnames

**Review by EliorKedar — APPROVED** [review]

**Pull request merged**